src/components/connection-prompt/ConnectDialogContainer.svelte

```diff
@@ -193,9 +230,10 @@
       {:else if currentStage === 'usb2'}
         <ConnectCableDialog
           {currentStage}
-          onSkipClick={() =>
-            ($connectionDialogState.connectionState =
-              ConnectDialogStates.CONNECTING_MICROBITS)}
+          onAltOrSkipClick={() => {
+            $connectionDialogState.connectionState = ConnectDialogStates.START_BLUETOOTH;
+            currentStage = 'usb';
+          }}
```

Comment:
It's a little difficult to imagine what the dialogs look (e.g. whether the `onAltOrSkipClick` prop is for the alternative button or skip button, whether the dialog has an alternative button or skip button) unless we go into the dialog to see whether the alt or skip button is shown depending on the `currentStage` prop. 

Maybe we can have a separate prop for each button (e.g. `onAltClick`  and `onSkipClick`) and render the alt or skip button in the dialog depending on whether the corresponding handler exists? 

Or better yet, maybe even pass in the dialog heading and subtitle into the dialog component so that the logic for how to render the dialog depending on whether it is `usb1`, `usb2`, or `usb` can occur in the same place, instead of having whether it is `usb1`, `usb2`, or `usb` logic on both, ConnectDialogContainer and ConnectCableDialog component. Hope this suggestion makes sense!

Reply:
Makes sense. Tidies the connect cable dialog and moves the mess into the already messy component. [1e5d22b](https://github.com/microbit-foundation/ml-trainer/pull/158/commits/1e5d22bd703992934441d11177022d2be5d75798)

Reply:
Haha yes, hopefully we get the opportunity to refactor ConnectDialogContainer at some point, but at least it's one step towards the right direction :)